SPECS-EXTENDED/python-betamax/python-betamax.spec

```diff
@@ -1,25 +1,20 @@
-%{!?python3_version: %define python3_version %(python3 -c "import sys; sys.stdout.write(sys.version[:3])")}
-%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-%{!?__python3: %global __python3 /usr/bin/python3}
-%{!?py3_build: %define py3_build CFLAGS="%{optflags}" %{__python3} setup.py build}
-%{!?py3_install: %define py3_install %{__python3} setup.py install --skip-build --root %{buildroot}}
-
 %global srcname betamax
 
 # tests need internet access therefore disabled by default
 # $ fedpkg mockbuild --enable-network --with=tests
 %bcond_with tests
 
-Summary:        VCR imitation for python-requests
 Name:           python-%{srcname}
-Version:        0.8.1
-Release:        12%{?dist}
-License:        ASL 2.0
+Version:        0.9.0
+Release:        1%{?dist}
```

Comment:
Increase the release to 4(+1 from changelog). This is what all your team is following.